steward/src/commands/keys/eth/delete.rs

```diff
@@ -3,21 +3,22 @@ use abscissa_core::{clap::Parser, Application, Command, Runnable};
 use signatory::FsKeyStore;
 use std::path;
 
-/// Delete Key
+/// Gorc keys eth delete [name]
```

Comment:
We should avoid referencing `gorc` inside steward since it's a separate CLI implementation. I also don't think this type of comment is necessary as running `--help` for a specific command implies the format already.